packages/gatsby-link/src/__tests__/rewrite-link-path.ts

```diff
@@ -1,11 +1,23 @@
-import { rewriteLinkPath } from "../rewrite-link-path"
+import type { TrailingSlash } from "gatsby-page-utils"
+
+import { rewriteLinkPath, type rewriteLinkPathType } from "../rewrite-link-path"
```

Comment:
```suggestion
import { rewriteLinkPath, rewriteLinkPathType } from "../rewrite-link-path"
```

We don't use the "inline type" syntax (or however you might want to call that)